logger/logger_test.go

```diff
@@ -26,13 +26,14 @@ import (
 )
 
 func TestLogger(t *testing.T) {
```

Comment:
Maybe add a test the verifies an err is returned from `(Get)LogFileLocation()`, like: `TestLogFileLocation()`, and uses some bogus value for `result` that can not succeed?

Ignore if this is a bad idea. I'd usually pull branches and try it myself, but the branch of branch...got a bit tricky, so just showing it out there.